frontend/app/src/App.tsx

```diff
@@ -1908,11 +1908,12 @@ export class App extends PureComponent<Props, State> {
             <StyledApp
               className={outerDivClass}
               data-testid="stApp"
-              data-teststate={
+              data-test-script-state={
```

Comment:
it'd be great if we had a unit test for these attributes as well instead of relying on expensive e2e tests for this

Reply:
Added some tests 👍